samples/DevTestLabs/GatewaySample/README.md

```diff
@@ -0,0 +1 @@
+NOTE: By using the sample templates, you agree to Remote Desktop Gateway�s terms. Click  to read [RD Gateway license terms](https://www.microsoft.com/en-us/licensing/product-licensing/products).
```

Comment:
Consider being a bit more elaborate in the description of these samples.

Reply:
Updated to add explanatory paragraph.